src/App.jsx

```diff
@@ -1,34 +1,18 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
 import './App.css'
 
 function App() {
-  const [count, setCount] = useState(0)
-
+const todos = [
```

Comment:
There is a small bug here. Your application is running fine. However, future issue will arise. Always have your dev tools open. This can give you a quick hint at problems that you've overlooked or didn't think to check.